components/_patterns/02-molecules/accordion-item/accordion-item.js

```diff
@@ -1,57 +1,51 @@
-// UNCOMMENT IF DRUPAL - see components/_meta/_01-foot.twig for attachBehaviors
+// UNCOMMENT NEXT 2 LINES IF DRUPAL - see components/_meta/_01-foot.twig for attachBehaviors
 // Drupal.behaviors.accordion = {
 //   attach: function (context, settings) {
 
-(function () { // REMOVE IF DRUPAL
-
-  'use strict';
-
+// REMOVE NEXT LINE IF DRUPAL
+(function accordion() {
   // Set 'document' to 'context' if Drupal
-  var accordionTerm = document.querySelectorAll('.accordion-term');
-  var accordionDef = document.querySelectorAll('.accordion-def');
+  const accordionTerms = document.querySelectorAll('.accordion-term');
+  const accordionDefs = document.querySelectorAll('.accordion-def');
 
   // If javascript, hide accordion definition on load
   function jsCheck() {
-    for (var i = 0; i < accordionDef.length; i++) {
-      if (accordionDef[i].classList) {
-        accordionDef[i].classList.add('active');
-        accordionDef[0].previousElementSibling.classList.add('is-active');
+    Array.from(accordionDefs).forEach((accordionDef) => {
```

Comment:
Since this anonymous function you're creating only accepts one parameter, you can remove the parenthesis around it:

```javascript
Array.from(accordionDefs).forEach(accordionDef => {

});
```

Reply:
Additionally, another way to do this:

```javascript
[...accordionDefs].map(accordionDef => {
  // ... etc.
});
```

Reply:
eslint is flagging the single parameter one saying it has to have parentheses when the  body has curly braces.

Reply:
Oh, gotcha. Makes sense!